src/sentry/tasks/scheduler.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import, division
+
+import logging
+
+from django.utils import timezone
+
+from sentry.app import locks
+from sentry.models import ScheduledJob
+from sentry.tasks.base import instrumented_task
+
+logger = logging.getLogger('sentry.scheduler')
+
+
+@instrumented_task(name='sentry.tasks.enqueue_scheduled_jobs')
+def enqueue_scheduled_jobs(**kwargs):
+    from sentry.celery import app
+
+    with locks.get('scheduler.process', duration=60).acquire():
+        job_list = ScheduledJob.objects.filter(
```

Comment:
I forget exactly how the query cache works but it'd be safer to at this point just coerce this result to a `list` so that way it's explicit that you don't intend for it to run twice.